app/views/brexit_checker/_action_audiences.html.erb

```diff
@@ -1,27 +1,40 @@
-<% if audiences.any? %>
-  <% audiences.each.with_index do |audience, index| %>
-    <% if audience[:actions].any? %>
-      <div class="govuk-grid-row govuk-!-margin-bottom-6">
-        <div class="govuk-grid-column-full">
-          <h2 class="govuk-heading-l action-audience__heading"><%= audience[:heading] %></h2>
-        </div>
-        <div class="govuk-grid-column-full">
-          <%= render "action_list", {
-            audience_heading: audience[:heading],
-            audience_index: index + 1,
-            items: audience[:actions]
-          } %>
-        </div>
-      </div>
+<% if audiences.any?  %>
+  <section class="brexit-checker-actions">
+    <% audiences.each do |audience| %>
+      <% if audience[:groups].any? %>
+        <header class="govuk-section__header">
+          <h2 class="govuk-heading-l action-audience__heading">
+            <%= audience[:heading] %>
+          </h2>
+        </header>
+        <% audience[:groups].each do |group| %>
+          <section class="brexit-checker-actions__group">
+            <div class="govuk-grid-row">
+              <div class="govuk-grid-column-one-third">
+                <%= render 'results_answers', answers: group[:criteria], heading: group[:heading] %>
```

Comment:
not sure about renaming criteria to answers. Why not leave them criteria - less to remember

Reply:
good point, have renamed this everywhere that previously referred to 'answers'